php_fpm/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM php:7.4-fpm
+
+ARG NAMELESSMC_VERSION=2.0.0-pr7
```

Comment:
Can we change this to the latest stable version, apparently 1.0.21, so I can merge this into master? Users can specify the build version optionally if they want to build v2. (can you add it to the README as well?)

Reply:
This has been made specifically for v2, I have not tested v1. v1 no longer receives development and v2 is nearing stable release. Judging from conversations in the discord server no one really uses v1 anymore. If you to wait until the full release though, you can mark this PR as a draft and we'll wait.